test/parallel/test-crypto-fips.js

```diff
@@ -24,7 +27,7 @@ const CNF_FIPS_OFF = fixtures.path('openssl_fips_disabled.cnf');
 let num_children_ok = 0;
 
 function compiledWithFips() {
```

Comment:
It looks like this function name no longer provides the correct description?
Perhaps just inline `compiledWithFips()` calls to `fipsMode` instead?

Reply:
I think the function name is still valid. The change is from only being able to detect if node was statically linked to a FIPS compatible OpenSSL library (`openssl_fips') to now also being able to determine if node was dynamically linked against as FIPS compatible OpenSSL library.

Reply:
@ChALkeR Just wanted to ask if you still would like to see this function renamed/inlined after my attempt to clarify above?

Reply:
ping @ChALkeR

Reply:
I believe @ChALkeR is ill add rightfully resting and not responding here. 
Would it be alright if I dismiss his review and offer to follow up with a separate PR if he feels strongly about these changes once he is feeling better?

Reply:
@danbev I just dismissed my review, sorry for the delay.

Reply:
@ChALkeR Thanks! Hope you are feeling better!